index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="ru">
+	<head>
+		<meta charset="UTF-8">
+		<title>Dzianis Kavalionak Resume</title>
+		<link rel="stylesheet" href="main.css">
+	</head>
+	<body>
+		<script src="zeros.js"></script>
+		<header>
+			<img class="header_bg" src="img/bg_photo.jpg" alt="s" width="1024" height="500">
+			<ul class="header_menu">
+				<li><a href="#Contacts">Контакты</a></li>
+				<li><a href="#example-tasks">Примеры работ</a></li>
+				<li><a href="#code1">Код</a></li>
+			</ul>	
+			<div class="header_about">
```

Comment:
`html/css`: You should one consolidated naming for classes `class-name` (kebab-case). camelCase for id.